conda-store-server/conda_store_server/worker/tasks.py

```diff
@@ -106,10 +106,15 @@ def task_update_storage_metrics(self):
 
 
 @shared_task(base=WorkerTask, name="task_cleanup_builds", bind=True)
-def task_cleanup_builds(self, build_ids: typing.List[str] = None, reason: str = None):
+def task_cleanup_builds(
+    self,
+    build_ids: typing.List[str] = None,
+    reason: str = None,
+    is_canceled: bool = False,
+):
     conda_store = self.worker.conda_store
     with conda_store.session_factory() as db:
-        build_cleanup(db, conda_store, build_ids, reason)
+        build_cleanup(db, conda_store, build_ids, reason, is_canceled)
```

Comment:
Test if this status also shows up and works in the new web UI.

Reply:
conda-store-ui will need to be updated after this lands to support this better. Currently, it shows undefined for these builds because it doesn't know about this new status:
<img width="397" alt="Screen Shot 2024-02-26 at 00 41 12" src="https://github.com/conda-incubator/conda-store/assets/10469885/84369c80-bbb2-4c0c-9916-0b33da23abd2">